test/integration/compliance_history_test.go

```diff
@@ -655,8 +655,8 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
 				g.Expect(err).ToNot(HaveOccurred())
 
 				g.Expect(events).To(
-					HaveLen(expectedEvents),
-					fmt.Sprintf("expected cluster %s to have %d events", cluster, expectedEvents),
+					HaveLen(2),
+					fmt.Sprintf("expected cluster %s to have %d events", cluster, 2),
```

Comment:
This should set "expectedEvents := 2" above this rather than changing the variable to a number.